checks/check_status.py

```diff
@@ -797,6 +798,17 @@ def body_lines(self):
                     "  Password: %s" % self.hidden_password
                 ]
 
+        if self.too_big_count:
+            if self.too_big_count > 1:
+                lines += [
+                    "%d messages were rejected by the server as too big." % self.too_big_count
```

Comment:
To stay consistent with the rest of the info page, I'd prefer this message to be displayed with the other stats on the transactions, i.e. something like:

```
Transactions rejected (too large): 15
```

right below `Transactions flushed`. What do you think?